backend/src/openarchiefbeheer/logging/logevent.py

```diff
@@ -29,6 +30,32 @@ def destruction_list_updated(destruction_list: DestructionList) -> None:
     _create_log(model=destruction_list, event="destruction_list_updated")
 
 
+def destruction_list_reassigned(
+    destruction_list: DestructionList,
+    assignees: list[DestructionListAssignee],
+    comment: str,
+    user: User,
+) -> None:
+    _create_log(
+        model=destruction_list,
+        event="destruction_list_reassigned",
+        user=user,
+        extra_data={
+            "assignees": [
+                {
+                    "user": {
+                        "pk": assignee["user"].pk,
+                        "email": assignee["user"].email,
+                        "username": assignee["user"].username,
+                    },
+                }
+                for assignee in assignees
+            ],
```

Comment:
In this way we only keep track of the new assignees. Do we want to also record the old ones?

Reply:
Based on the history the old ones can be determined? I'm not against this idea but I do like the JSON data to be a subset of the actual object, adding the old ones would pollute this.

Reply:
I see. But i don't think that the original assignees are logged anywhere, so maybe that needs to be added as extra data to the `destruction_list_created` log?

Reply:
I've added it to to destruction_list_created